lib/driver.js

```diff
@@ -844,6 +844,7 @@ class XCUITestDriver extends BaseDriver {
     let maxTypingFrequency = util.hasValue(this.opts.maxTypingFrequency) ? this.opts.maxTypingFrequency : 60;
     let shouldUseSingletonTestManager = util.hasValue(this.opts.shouldUseSingletonTestManager) ? this.opts.shouldUseSingletonTestManager : true;
     let shouldUseTestManagerForVisibilityDetection = false;
+    let wdaEventloopIdleDelay = this.opts.eventloopIdleDelaySec || 0;
```

Comment:
`let eventloopIdleDelaySec = this.opts. wdaEventloopIdleDelay || 0` and `eventloopIdleDelaySec` as a desired capability in create session below?